x-pack/plugins/uptime/public/components/functional/monitor_list.tsx

```diff
@@ -124,7 +124,7 @@ export const MonitorListComponent = ({
                 locationName
               ) : (
                 <EuiLink
-                  href="https://www.elastic.co/guide/en/beats/heartbeat/current/add-host-metadata.html#add-host-metadata"
+                  href="https://www.elastic.co/guide/en/beats/heartbeat/current/add-observer-metadata.html"
```

Comment:
This should actually point to the page that will go live with the merge of this PR https://github.com/elastic/beats/pull/12433 I'm realizing . The URL would be: https://www.elastic.co/guide/en/beats/heartbeat/current/configuration-observer-options.html

It would go live with 7.2

Reply:
Ok - is there any good way to test that this works before we merge and backport to 7.2?